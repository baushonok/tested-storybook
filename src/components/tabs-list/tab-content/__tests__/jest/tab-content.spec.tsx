import React from 'react';
import renderer from 'react-test-renderer';

import TabContent from '../..';

describe('TabContent', () => {
  describe('should match snapshot', () => {
    const content = 'some content';

    test('when active', () => {
      const tree = renderer
        .create(<TabContent data={content} isActive />)
        .toJSON();

      expect(tree).toMatchSnapshot();
    });

    test('when inactive', () => {
      const tree = renderer.create(<TabContent data={content} />).toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
