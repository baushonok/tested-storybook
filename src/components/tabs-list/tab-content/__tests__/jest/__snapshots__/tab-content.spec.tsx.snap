// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabContent should match snapshot when active 1`] = `
<section
  className="tab-content "
>
  some content
</section>
`;

exports[`TabContent should match snapshot when inactive 1`] = `
<section
  className="tab-content tab-content_hidden"
>
  some content
</section>
`;
